packages/mystmd-py/setup.cfg

```diff
@@ -0,0 +1,29 @@
+[metadata]
+name = mystmd
+version = 0.0.6
```

Comment:
Once this PR is approved, we should do one more release to get this current with the npm version.